src/components/Logout.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import decoration from '../assets/Decoration.svg';
+import { NavLink } from 'react-router-dom';
+import Header from './Header';
+
+const Logout=()=>{
+
+    return<>
+        <section className='logout'>
+            <div className='headerWindow'>
+                <Header/>
+            </div>
+            <div className='logContainer'>
+                <div className='logWindow'>
+                    <h1>Wylogowanie nastąpiło pomyślnie!</h1>
+                    <img className='logoutImage' src={decoration} alt='decoration'/>
+                    <NavLink to="/"><button className='logoutBtn'>Strona główna</button></NavLink>
+                </div>
+            </div>
+        </section>
+    </>   
+}
+
+export default Logout;
```

Comment:
This is somewhat controversial but i like using named exports rather than default exports more, as you have to import it with exactly the same name as it is being exported